released/usecases/kyma_devcontainer_runtime/version.tf

```diff
@@ -0,0 +1,8 @@
+terraform {
+  required_providers {
+    btp = {
+      source  = "SAP/btp"
+      version = "1.0.0-rc1"
+    }
+  }
+}
```

Comment:
I think it makes sense to add at least a comment on the provider configuration as in the other scenarios e.g.:

```terraform
# Please checkout documentation on how best to authenticate against SAP BTP
# via the Terraform provider for SAP BTP
provider "btp" {
  globalaccount = "<YOUR GLOBALACCOUNT SUBDOMAIN>"
}
```

Reply:
Ok, but I would put commented out, because otherwise it can't be consumed as module.